services/core-web/src/tests/mocks/keycloakMocks.js

```diff
@@ -0,0 +1,14 @@
+export const getKeycloakMock = (mockInitialized=true, mockAuthenticated=true, mockClient_roles=['role_view']) => {
```

Comment:
So far seems to only work in the index test file? AuthenticationGuard test file broke when imported